example/deep-embedded-clustering/model.py

```diff
@@ -22,7 +22,7 @@
 import numpy as np
 try:
     import cPickle as pickle
```

Comment:
why use cPickle? why not just pickle. In fact cPickle is not used in python3. most functions of cPickle is there in pickle

Reply:
This is correct usage...  We first try to use __cPickle__ which is compiled C code in Python 2 and is nice and fast but if we fail to import that (because it is not pip installed on Python 2 or because we are running on Python 3) then we use the normal standard library pickle.